components/automate-ui/scripts/build.sh

```diff
@@ -17,7 +19,7 @@ if [ $? -ne 0 ]; then
 fi
 
 # If no errors, next check for any warnings.
-echo "$out" | grep 'WARN:' >/dev/null
+echo "$out" | grep "$WARNING_LABEL" >/dev/null
```

Comment:
[nit]
```suggestion
grep -q "$WARNING_LABEL" <<<"$out"
```
But it really doesn't matter 😃

Reply:
I like that; thanks!